cmd/containerboot/tailscaled.go

```diff
@@ -166,3 +169,64 @@ func tailscaleSet(ctx context.Context, cfg *settings) error {
 	}
 	return nil
 }
+
+func watchTailscaledConfigChanges(ctx context.Context, path string, lc *tailscale.LocalClient, errCh chan error) {
+	var (
+		tickChan          <-chan time.Time
+		tailscaledCfgDir  = filepath.Dir(path)
+		prevTailscaledCfg []byte
```

Comment:
I guess we'll get a single spurious reload + log line at startup? It might be nice to suppress that if it's not too difficult just to make sure the logs aren't misleading.